### DISCUSSION TITLE
Help. Why are my days stretching only on certain months?

### DISCUSSION THREAD

**Discussion by bvelasquez** [gpbl/react-day-picker#1261] [Support]
Purposely styled the component to illustrate the issue I'm having. The days are stretched only on certain months. October looks correct, but Aug, and Sept. are stretched. No additional styling except for the color changes.

I did notice that it only happens when numberOfMonths is greater than 1.

Edit: Ok, October has 6 rows of data. The other two months only 5 rows of data. So, they are stretching to maintain size with the October table.  any way to correct this?

![2021-10-05_14-47-48](https://user-images.githubusercontent.com/2302210/136107847-0451414a-a03e-410f-bf04-acae7e8cfbe2.png)

**Comment by Kosai106** [comment]
(Not an answer but didn't feel like creating a new discussion/issue is necessary)

I came to try and find a solution to this exact thing.

In our case it's not a big deal but it does look weird and I'd love for a way to opt out of this behaviour. 

<img width="820" alt="Screenshot 2021-12-16 at 11 47 57" src="https://user-images.githubusercontent.com/6379824/146357765-87516002-7baa-49a8-b586-69de5d2a29f5.png">

**Comment by tjfred35** [comment]
The CSS reason for this is:
```
.DayPicker-Months {
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
}
```

flexbox is stretching vertically by default. Changing it to this makes it look like this instead:

```
.DayPicker-Months {
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
    align-items: start; /** ADD THIS HERE */
}
```

<img width="520" alt="Screen Shot 2021-12-16 at 7 43 59 AM" src="https://user-images.githubusercontent.com/2838338/146374361-1d64af28-c20f-4167-980b-af97886d2965.png">

https://codesandbox.io/s/react-day-picker-examples-modifiers-forked-9m4ih?file=/example.js

**Reply by Kosai106** [reply]
Oh man, you are amazing. 👏 
I've been trying a bunch of different things already on the `.DayPicker-Day` class, I hadn't considered targeting the wrapper.
